### DISCUSSION TITLE
Warning: Calling `Tap.names` is deprecated! There is no replacement.

### DISCUSSION THREAD

**Discussion by iAGorynT** [Homebrew/discussions#5365] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.0
ORIGIN: https://github.com/Homebrew/brew
HEAD: 8378cc825d83acffd125fb0fec041793df378a57
Last commit: 10 hours ago
Core tap JSON: 14 May 14:14 UTC
Core cask tap JSON: 14 May 14:14 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 12
Homebrew Ruby: 3.1.4 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: dodeca-core 64-bit cometlake
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.6.0 => /usr/bin/curl
macOS: 14.5-x86_64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Since updating MacOS to version 14.5, I receive the following message when issuing the `brew autoupdate version` and `brew autoupdate start` commands:

Warning: Calling `Tap.names` is deprecated! There is no replacement.
Please report this issue to the Homebrew/homebrew-autoupdate tap:
  /usr/local/Homebrew/Library/Taps/homebrew/homebrew-autoupdate/lib/autoupdate/core.rb:32

Any insights / suggestions  would be appreciated.

**Comment by Bo98** [comment, marked as answer]
This is an issue that should be reported to https://github.com/DomT4/homebrew-autoupdate

**Reply by iAGorynT** [reply]
Thank you!
